doc/source/development/contributing_environment.rst

```diff
@@ -228,6 +228,14 @@ To compile pandas with meson, run::
    # If you do not want to see this, omit everything after --no-build-isolation
    python -m pip install -ve . --no-build-isolation --config-settings editable-verbose=true
 
+.. note::
+   The version number is pulled from the latest repository tag. Be sure to fetch the latest tags from upstream
+   before building::
```

Comment:
Can you use `code-block:: shell` instead of `::`?

Reply:
@mroeschke FWIW, the code block just above with the install command also uses `::`, so I think the current version is fine here to keep it consistent. 
(if we want to generally move to use `code-block:: shell`, it might be better to first do a PR to change it consistently in the full file?)

Reply:
Okay sure this can be a follow up